common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -82,7 +82,11 @@ public static String modifyMultipleMatchKeyword(String query) {
                     if (key.length() < 3) {
                         temp.add(key);
                     } else {
-                        temp.add(new String("+" + key + "*"));
+                        if (purpose.equalsIgnoreCase("survey_counts")) {
```

Comment:
Please add a comment - I don't understand why we would only glob for survey counts

Reply:
Added detail in the description.